server/store/kvstore/poll.go

```diff
@@ -78,6 +80,8 @@ func (s *PollStore) Update(prev *poll.Poll, new *poll.Poll) error {
 
 // Delete deletes a poll from the KV Store.
 func (s *PollStore) Delete(poll *poll.Poll) error {
+	//nolint:revive // Ignore linter error here, because "model.AppError(nil) != nil" always returns true.
```

Comment:
What is the concrete error? I'm unable to reproduce it locally.

Reply:
Without this comment, I encountered the following lint error.

```shell
$ go version
go version go1.21.4 linux/amd64
$ golangci-lint version
golangci-lint has version v1.52.2 built with go1.20.2 from (unknown, mod sum: "h1:FrPElUUI5rrHXg1mQ7KxI1MXPAw5lBVskiz7U7a8a1A=") on (unknown)

$ make check-style
...
Running golangci-lint
golangci-lint run ./...
server/store/kvstore/poll.go:83:2: if-return: redundant if ...; err != nil check, just return error instead. (revive)
        if err := s.api.KVDelete(pollPrefix + poll.ID); err != nil {
                return err
        }
make: *** [Makefile:52: check-style] Error 1
```

Reply:
After updating golangci-lint 1.55.2, the lint error is not detected without comments, so I removed comments to avoid lint errors.